src/backend/base/langflow/graph/graph/runnable_vertices_manager.py

```diff
@@ -68,14 +73,14 @@ def remove_from_predecessors(self, vertex_id: str):
             if vertex_id in self.run_predecessors[predecessor]:
                 self.run_predecessors[predecessor].remove(vertex_id)
 
-    def build_run_map(self, graph):
+    def build_run_map(self, predecessor_map, vertices_to_run):
```

Comment:
Would it be helpful to add type? 
predecessor_map:  Dict[str, list[str]], verties_to_run: set